src/asmCrashReport.cpp

```diff
@@ -413,7 +430,10 @@ namespace asmCrashReport
    void _posixSetupSignalHandler()
    {
       // setup alternate stack
-      stack_t ss{ static_cast<void*>(sAlternateStack), SIGSTKSZ, 0 };
+      stack_t ss;
```

Comment:
I don't mind this change - just curious - does gcc not like the initializer list?

Reply:
> does gcc not like the initializer list?
Different versions libc on Linux have different ordering =(